app/src/test/java/com/github/sdp_begreen/begreen/entities/PostTest.kt

```diff
@@ -0,0 +1,23 @@
+package com.github.sdp_begreen.begreen.entities
+
+import android.graphics.Bitmap
+import org.junit.Assert.assertEquals
+import org.junit.Test
+import org.mockito.Mockito.mock
+
+class PostTest {
+
+    private val mockBitmap = mock(Bitmap::class.java)
```

Comment:
I used a mock to check that the image getter returns the expected bitmap. Should I change it to a fake value ? I though that the purpose of the mock library was automatically prevent this fake instances.  

bitmap = Bitmap.createBitmap(100, 100, Bitmap.Config.ARGB_8888)

Reply:
Anyway, I've changed because I was not confortable with the mock. I don't know was it was underlined in red even if the tests passed. But I am still curious about why you suggested to change it